.github/workflows/closeDevIssues.yml

```diff
@@ -0,0 +1,21 @@
+name: Close PR on develop merge
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  close_issues:
+    if: github.event.pull_request.merged == true && github.base_ref == 'develop'
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@v4
+      
+    - name: Run script
+      working-directory: .github/workflows/scripts
+      run: |
+        pip install requests
+        python closeDevIssues.py ${{ secrets.GITHUB_TOKEN }} ${{ github.event.pull_request.number }}
```

Comment:
Maybe add owner & repo as parameters as well :) Then, we can recycle the action better :)